pandas/core/resample.py

```diff
@@ -840,7 +839,90 @@ def interpolate(
     ):
         """
         Interpolate values according to different methods.
+
```

Comment:
Can you document all the parameters too?

Reply:
Done.